### DISCUSSION TITLE
do we really need a `chemical_entity_set`?

### DISCUSSION THREAD

**Discussion by turbomam** [microbiomedata/nmdc-schema#2148] [General]
If we can find one or more namespaces that unambiguously identify reagents and metabolites, with high fidelity regarding the forms of those chemicals, then we don't need a `chemical_entity_set`. We just need slots that mention the external identifiers. Conventionally, those would be web-resolvable CURIes.

**Comment by kheal** [comment]
@kheal  Tagging myself so I get updates.

**Comment by kheal** [comment]
we've removed `chemcial_entity_set` - going to close the discussion
